client/src/modules/stock/lots/registry.service.js

```diff
@@ -154,6 +154,21 @@ function LotsRegistryService(uiGridConstants, Session) {
     <div>
       <b>{{ grid.appScope.countGridRows() }}</b>
       <span translate>TABLE.AGGREGATES.ROWS</span>
+
+      <span class="fa fa-circle icon-expired"></span>
```

Comment:
This is a really good idea @lomamech.  You found the perfect place to put this.

Reply:
See if the tests pass now

bors try